modules/javafx.graphics/src/main/java/javafx/scene/layout/package-info.java

```diff
@@ -195,5 +195,19 @@
  * that button.  If an application wishes to have the effect, clip, or transform
  * factored into the layout of a node, it should wrap that node in a Group.
  * </p>
+ *
+ *
+ * <h2>Layout Orientation</h2>
+ * The layout orientation of the scene graph is determined by the {@link javafx.scene.Scene#nodeOrientationProperty() Scene.nodeOrientation}
+ * and {@link javafx.scene.Node#nodeOrientationProperty() Node.nodeOrientation} properties.
+ * It is effectively left-to-right by default, but can be right-to-left depending on the locale of the operating system.
+ * If a layout orientation is set on the scene or on any node of the scene graph, descendants of the scene or node will
+ * inherit the specified orientation.
+ * <p>
+ * For layout containers, the layout orientation determines the visual order of their children. If a layout container
+ * has named areas, the names always retain their default meaning. For example, the {@code left} and {@code right}
```

Comment:
It might be just me, but I still find this explanation very confusing (what is the default meaning of left?).

Could we just say that for historical reasons, "left" and "right" in reality mean "leading" and "trailing", or maybe that in the RTL mode the left/right are swapped, with the "left" nodes appearing on the leading (visually right) side, and the "right" ones are on the trailing (visually left) side?

Also, do you think we should explicitly mention entities where this logic applies  (`BorderPane`, `HeaderBar`)?

Reply:
I've reverted the changes to this file.